docs/reference/search/retriever.asciidoc

```diff
@@ -250,7 +250,7 @@ GET /restaurants/_search
           }
         }
       ],
-      "rank_constant": 0.3, <5>
+      "rank_constant": 1, <5>
```

Comment:
Is it possible to add tests so that we don't have an error here again?

Reply:
Good point, it's not clear to me why the original docs skipped tests. But in this case the test wouldn't have errored out in any case. I'll create an issue to add tests to this doc and seek your input @kderusso in follow up work.